byzcoin/README.md

```diff
@@ -14,6 +14,43 @@ them as seem fit.
 This document describes the part of ByzCoin that are implemented and how to
 use them. It should grow over time as more parts of the system are implemented.
 
+## Short summary of ByzCoin
+
+- ByzCoin is a **distributed ledger**: a database that is
+  distributed across several nodes, and where the authority to add changes
+  to the database is decentralized.
+- To ensure that updates to the database are strictly ordered, ByzCoin
+  uses a **blockchain** called a **Skipchain**: every node has its local copy of the ledger
+  (database). Every change to the ledger is packed into a block that is
+  cryptographically chained to the previous one, and forward links in
+  the chain represent consensus among the verifiers that these changes
+  belong in the database.
+- In ByzCoin, the data are organised as instances of **Smart
+  Contracts**, where a smart contract can be seen as a class and an
+  instance an object of this class. In ByzCoin, one can *Spawn* a new
+  instance of a contract, *Invoke* a command (or method) on an existing
+  instance, or *Delete* an instance. *Spawn*, *Invoke*, and *Delete*
+  represent all the possible actions a user can do to update the ledger.
+- Every request to update the ledger is a transaction made up of one
+  or more instructions. The transactionn is sent to one of the nodes.
+  All instructions in the transaction must be approved by a quorum of
+  the nodes, or else the entire transaction is refused.
+- As an example, we can have a simple *Project* smart contract that has
```

Comment:
I would not put the example in the `Short summary`, but rather in it's own `## Example`

Reply:
We should then provide a more complete example. To keep it simple I prefer if we keep it there.